### PR TITLE
feat(gatsby-build): Added gatsby-build pack

### PR DESCRIPTION
**Pull request by rudouglas**
The template pack allows you to get visibilility into the build time of your Gatsby Site,
  using Open Telemetry to collect each step as Trace Data

### CODE REVIEW

**Comment by rudouglas** [comment]
gatsby-build

File: packs/gatsby-build/config.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+# Name of the pack
+name: Gatsby Build
+
+# Description of the pack
+description: |+
+  The template pack allows you to get visibilility into the build time of your Gatsby Site,
```
Comment (polfliet) [inline packs/gatsby-build/config.yml:7]: Suggestion + typo fix: `The Gatsby observability pack allows you to get visibility into the build time of your Gatsby Site,`

File: packs/gatsby-build/config.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  using Open Telemetry to collect each step as Trace Data
```
Comment (polfliet) [inline packs/gatsby-build/config.yml:8]: Is there a little bit more info we could provide? Like what are these steps?
`as Trace Data` -> `as spans in a distributed trace` ?

File: packs/gatsby-build/config.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# Support level: New Relic | Verified | Community
+level: New Relic
+
+# Design
+icon: icon.jpeg
```
Comment (polfliet) [inline packs/gatsby-build/config.yml:14]: can you add a Gatsby logo & icon?

File: packs/gatsby-build/config.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+logo: logo.png
+
+# Authors of the pack
+authors:
+  - New Relic
+  - Ruairi Douglas
+
+references:
```
Comment (polfliet) [inline packs/gatsby-build/config.yml:22]: We should remove the references as this is not relevant for this pack, right?

**Review by polfliet — CHANGES_REQUESTED** [review]
Such an awesome contribution which would be great to get merged in!

I have left some comments on the PR but please also review the guidelines as defined here: https://github.com/newrelic/newrelic-observability-packs/issues/46

File: packs/gatsby-build/config.yml (hunk 1 above)
Comment (rudouglas) [reply packs/gatsby-build/config.yml:7]: though visibilility is a nice sounding word 😂

File: packs/gatsby-build/config.yml (hunk 2 above)
Comment (rudouglas) [reply packs/gatsby-build/config.yml:8]: Ye these should be as spans in a DT

File: packs/gatsby-build/config.yml (hunk 4 above)
Comment (rudouglas) [reply packs/gatsby-build/config.yml:22]: ye I had actually removed these but hadn't pushed it, they're not relevant for this instance

File: packs/gatsby-build/config.yml (hunk 3 above)
Comment (rudouglas) [reply packs/gatsby-build/config.yml]: done :)

**Comment by rudouglas** [comment]
All changes made @polfliet  🙇‍♂️

**Review by polfliet — APPROVED** [review]

**Comment by polfliet** [comment]
Thank you!

**Pull request merged**